server/matchmaker/matchmaker_queue.py

```diff
@@ -93,7 +93,7 @@ def __init__(
     def find_matches(self) -> None:
         self._logger.info("Searching for matches: %s", self.queue_name)
 
-        # Call self.match on all matches and filter out the ones that were canceled
+        # Call self.match on all matches and filter out the ones that were cancelled
```

Comment:
WOW! Never thought, that I will review american vs british grammar :-), ofc British ftw :-D :+1: